cookbooks/convection-box.prm

```diff
@@ -101,7 +101,9 @@ end
 
 subsection Formulation
   set Formulation = custom
-  set Temperature density approximation = adiabatic
+  set Mass conservation    = incompressible
```

Comment:
Oh, did we merge something that worked on an earlier version of the branch, but no longer?